website/docs/developer/deployment-guide/deployment-with-cumulus.md

```diff
@@ -246,6 +253,7 @@ Below describes the type of value expected for each variable.
 * `orca_default_bucket` (string) - default S3 glacier bucket to use for ORCA data.
 * `db_admin_password` (string) - password for the postgres user.
 * `db_host_endpoint`(string) - Database host endpoint to connect to.
+* `orca_restore_retrieval_type`(string) -The Tier for the restore request. Valid values are 'Standard'|'Bulk'|'Expedited'.
```

Comment:
This is not a required variable. It should be an optional variable and we should default to standard.

Reply:
updated